ISISComputingGroup/ibex_gui
# Ticket1125 device screens dialog

### Description of work
https://github.com/ISISComputingGroup/IBEX/issues/1125

This adds a dialog for configuring device screens and saving them.

### To test

Open the editor and create/modify some screens and hit "OK" to save.
Confirm that the changes are saved to the screens.xml file in the configuration directory.

**The set default button does nothing at the moment - it requires information relating to the other tickets**

Check the GET_SCREENS PV:
```caget -S -t %MYPVPREFIX%CS:BLOCKSERVER:GET_SCREENS | uzhex```

For my system the PV value doesn't change - I don't know if this is my machine. If yours also doesn't update then it must be a bug. In that case, please create a ticket along the following lines:

```
Blockserver: GET_SCREENS PV does not update when setting screens
```

### Acceptance criteria

Can create and modify the devices screens via the GUI.

 **Before merging we need to disable the menu option as we don't want the users playing with it until the other hackathon tickets are ready.** 

---

#### Code Review


### Functional Tests


### Final Steps
